src/Controller/Admin/BlogController.php

```diff
@@ -160,7 +161,7 @@ public function edit(Request $request, Post $post): Response
      */
     public function delete(Request $request, Post $post): Response
     {
-        if (!$this->isCsrfTokenValid('delete', $request->request->get('token'))) {
+        if (!$this->isCsrfTokenValid(PostVoter::DELETE, $request->request->get('token'))) {
```

Comment:
usage is right, but I don't know if we want here to use the original string or the class constant

doc is https://symfony.com/doc/3.4/controller/csrf_token_validation.html, but imho the doc is not clear about what a developer should use as `token_id`

Reply:
you use what you want, as long as you use the same id when validating the token here and when generating it in the template

Reply:
ah ok it has nothing to do with PostVoter then.. I'll revert here